api/handlers.go

```diff
@@ -943,30 +943,125 @@ func handleCollection(db *sql.DB, r ItemRequest) (*Collection, *HttpError) {
 }
 
 func handleLatestContentUnits(db *sql.DB, r BaseRequest) ([]*ContentUnit, *HttpError) {
-
-	// CU ids query
-	const query = `SELECT DISTINCT ON (type_id) id
-FROM content_units
-WHERE secure = 0 AND published IS TRUE
-ORDER BY type_id, (coalesce(properties ->> 'film_date', created_at :: TEXT)) :: DATE DESC, created_at DESC;`
+	const query = `
+WITH CUs AS (
+    SELECT row_number()
+           OVER (PARTITION BY type_id ORDER BY coalesce(properties ->> 'film_date', created_at :: TEXT) :: DATE DESC) AS row_number,
+           name, uid, content_units.id, coalesce(properties ->> 'film_date', created_at :: TEXT) :: DATE AS film_date
+    FROM content_units
+    LEFT OUTER JOIN content_types ct ON content_units.type_id = ct.id
```

Comment:
We can drop this join if we use IDs.
Make it a parameter and pass it an hard coded array of IDs using [CONTENT_TYPE_REGISTRY](https://github.com/Bnei-Baruch/archive-backend/blob/master/mdb/registry.go#L17).
This goes the same for the collections subquery.